ManageIQ/manageiq-appliance_console
# Add net-ssh gems needed to support ed25519 cipher

Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1815568

net-ssh requires the following gems for SSH ed25519 cipher support:
- pbnacl (>= 3.2, < 5.0)
- pbnacl-libsodium
- berypt_pbkdf (>= 1.8, < 2.8)

This PR adds the needed gems.

**To test:**

_Gen the ed25519 key:_
> miq-host> ssh-keygen -o -a 256 -t ed25519 -C "$(hostname)-$(date +'%d-%m-%Y')"

This produces:
  ~/.ssh/id_ed25519
  ~/.ssh/id_ed25519.pub

To use this put the content of the ~/.ssh/id_ed25519.pub in the ~/.ssh/authorized_keys on the remote host

_Then ensure the following command runs successfully:_

> miq-host> appliance_console_cli --fetch-key=_remote hostname or IPAddr_
